jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -1,104 +1,121 @@
 ---
-layout: classic-docs
-title: Config Policies
-categories: []
-description: How to leverage CircleCI containers
+description: Open preview of config policy management for CircleCI
 version:
 - Cloud
-- Beta
 ---
+= Config policy management overview
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
 
-## Introduction
-{: #introduction }
+TIP: Config policy management is currently in **open preview**. All aspects of the feature are subject to change. **We recommend you do not use this feature in production**.
 
-CircleCI uses `config.yml` files to define CI pipelines at the project level. For most use cases this is a convenient way of developing and iterating quickly as each pipeline can be made to meet the needs of the project as it grows. However, from an organizational standpoint, it can be tricky to manage and enforce organization-wide conventions and security policies. 
+== Introduction
 
-Config Policy Management uses a decision engine leveraging OPA (Open-Policy-Agent) to allow users to specify policies, and return a decision about if a Pipeline's Config complies with those policies. In the strictest case, if a pipeline configuration does not comply with the organization's policies, that pipeline will be blocked from running until it does comply. Decision's are also stored and can be audited allowing for useful feedback about what pipeline defintions are being run in your organization.
+Config policy management allows you to create policies for governing project configurations. In its strictest case, if a project configuration does not comply with the rules set out in the associated policy, that pipeline will be blocked from being triggered until it does comply.
```

Comment:
@sagar-connect does this language seem right? "pipeline will be blocked from being triggered" I want us to be very specific in the language we're using rather than saying "the build will fail".